internal/privy/encrypter.go

```diff
@@ -0,0 +1,62 @@
+package privy
+
+import (
+	"crypto/rand"
+	"crypto/sha256"
+	"encoding/hex"
+
+	"github.com/agl/gcmsiv"
```

Comment:
Since it's unused, let's comment out this file so we're not adding a new dependency.

Reply:
okiee it was used in private_data_test.go which is good to make sure we're going back and forth the right way but we can add it back in later